src/index.ts

```diff
@@ -15,6 +15,27 @@ import {validate} from './validator'
 
 export {EnumJSONSchema, JSONSchema, NamedEnumJSONSchema, CustomTypeJSONSchema} from './types/JSONSchema'
 
+/**
+ * Defines the available options for how enums can generated
+ */
+export const EnumGenType = {
+  Default: 'default',
+  /**
+   * Prepend enums with [`const`](https://www.typescriptlang.org/docs/handbook/enums.html#computed-and-constant-members)?
```

Comment:
I think you probably want to link to this?
https://www.typescriptlang.org/docs/handbook/enums.html#const-enums

Reply:
This was copied from the then-existing `enableConstEnums` flag documentation, but gladly updated! 🙌🏻